packages/apputils-extension/src/index.ts

```diff
@@ -498,12 +513,21 @@ const state: JupyterLabPlugin<IStateDB> = {
         // Maintain the query string parameters but remove `reset`.
         delete query['reset'];
 
+        const silent = true;
+        const hard = true;
         const url = path + URLExt.objectToQueryString(query) + hash;
         const cleared = commands.execute(CommandIDs.recoverState)
           .then(() => router.stop); // Stop routing before new route navigation.
 
         // After the state has been reset, navigate to the URL.
-        cleared.then(() => { router.navigate(url, { silent: true }); });
+        if (clone) {
+          cleared.then(() => { router.navigate(url, { silent, hard }); });
```

Comment:
In this branch is the splash not disposed?

Reply:
That's right. It's not disposed because the `hard` navigation means a reload.

Reply:
The visual effect I was going for was the splash page still existing as the page unloads. If you look closely, you'll see the splash start over after the refresh, but I think it's still less jarring than the splash disappearing and then reappearing.